ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2

```diff
@@ -149,6 +149,11 @@ host_monitoring_cron:
   minute: "*/5"
   job: ops-runner -f -s 15 -n csnm.network.int /usr/bin/cron-send-network-metrics
 
+- name: report rpm versions daily
+  hour: "1"
+  minute: "0"
+  job: ops-runner -f -s 120 -n csdov.rpm.versions /usr/bin/cron-send-docker-oc-versions
```

Comment:
Yeah, we have this for some things (like kernel version).

Here's an example:
https://github.com/openshift/openshift-tools/blob/prod/ansible/roles/os_zabbix/vars/template_os_linux.yml#L71-L74


Check some of the PCP items for more details.